core/src/main/java/bisq/core/locale/CurrencyUtil.java

```diff
@@ -502,6 +506,114 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         return currencies;
     }
 
+    public static List<TradeCurrency> getAllInteracETransferIdCurrencies() {
```

Comment:
Yes, I agree that it would make more sense to move this information closer to the Domain. I didn't spend too much time yet on bigger refactorings to prevent too many unexpected side effects by changing too much. But yes, I think moving it the PaymentMethod would make sense. Although it is a 1:1 relationship all the time AFAIK, but I think it would be a better fit for the PaymentMethod, which is used by the Payment Account.

Reply:
After looking at the code, I revert my opinion and also think adding it to the concrete account implementation is the better way to do this.